k8s/operators/src/pool/main.rs

```diff
@@ -146,8 +154,8 @@ impl OperatorContext {
 
 impl ResourceContext {
     /// Called when putting our finalizer on top of the resource.
-    #[tracing::instrument(fields(name = ?dsp.name_any()))]
-    pub(crate) async fn put_finalizer(dsp: Arc<DiskPool>) -> Result<Action, Error> {
+    #[tracing::instrument(fields(name = ?_dsp.name_any()))]
+    pub(crate) async fn put_finalizer(_dsp: Arc<NewDiskPool>) -> Result<Action, Error> {
```

Comment:
Why is the name changed to `_dsp`?

Reply:
We dont use dsp in method anywhere. Its used in tracing only in line 157. So marked it with _ .